### PR TITLE
mentors: Add mentors data structure

### LINKED ISSUE #18: Add mentors in Mentors tab

### PR DESCRIPTION
**Pull request by hemangsk**
cc @sils 
Fixes #18

### CODE REVIEW

File: data/mentors.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+mentors = [
+	{
+		"name" : "Mentor 1",
```
Comment (sils) [inline data/mentors.js]: can't we extract all the mentor data from the projects? I.e. go through all projects, take their github handle, get the bio from github, get the avatar from github and the name?

Comment (hemangsk) [inline data/mentors.js]: yeah that'll be much better. can we have a mentors team? that'll be more neat

**Comment by sils** [comment]
ed0128d is read

**Comment by sils** [comment]
ed0128d is ready

**Pull request merged**